quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
-    Installed path: /Users/user/.tiup/bin/tiup
-    ===============================================
-    Have a try:     tiup playground
-    ===============================================
-    ```
+You just started to use a fully functional distributed database cluster with:
 
-    Note the Shell profile path in the output above. You need to use the path in the next step.
+- **TiDB Server:** SQL layer that processes queries and speaks MySQL protocol
+- **TiKV:** Distributed storage layer that replicates data automatically and handles transactions across nodes
+- **PD (Placement Driver):** Cluster orchestrator that works behind the scenes to coordinate TiDB, TiKV, and TiFlash components, managing cluster metadata and ensuring data is distributed optimally
 
-2. Declare the global environment variable:
+Additionally, `tiup playground` installed optional components:
 
-    > **Note:**
-    >
-    > After the installation, TiUP displays the absolute path of the corresponding Shell profile file. You need to modify `${your_shell_profile}` in the following `source` command according to the path.
+- **TiFlash:** Real-time analytics engine that accelerates complex queries and reporting without slowing down your live application
+- Built-in **monitoring stack** with:
+  - TiDB Dashboard (default: http://127.0.0.1:2379/dashboard, user: `root`, no password)
+  - Grafana + Prometheus (default: http://127.0.0.1:3000, user: `admin`, password: `admin`)
 
-    {{< copyable "shell-regular" >}}
+You can verify your running database cluster components with:
 
-    ```shell
-    source ${your_shell_profile}
-    ```
+```shell
+tiup playground display
+```
 
-3. Start the cluster in the current session:
+## Developer workflow
 
-    - To start a TiDB cluster of the latest version with 1 TiDB instance, 1 TiKV instance, 1 PD instance, and 1 TiFlash instance, run the following command:
+The previous steps gave you an ephemeral environment - perfect for first experiments but everything is lost when you stop the playground. Let's set up a more suitable environment for development.
 
-        {{< copyable "shell-regular" >}}
+### Persistent playground environments
 
-        ```shell
-        tiup playground
-        ```
+Create a tagged playground that persists data between restarts:
 
-    - To specify the TiDB version and the number of instances of each component, run a command like this:
+```shell
+tiup playground --tag dev-env
+```
 
-        {{< copyable "shell-regular" >}}
+The `--tag` flag preserves your data after the playground environment restarts, and lets you run multiple environments using different tags (e.g., `dev-env`, `sample-app-env`).
 
-        ```shell
-        tiup playground v8.4.0 --db 2 --pd 3 --kv 3
-        ```
+Tags also makes cleanup explicit with:
 
-        The command downloads a version cluster to the local machine and starts it, such as v8.4.0. To view the latest version, run `tiup list tidb`.
+```shell
+tiup clean dev-env
+```
 
```

Comment:
```suggestion
```

Reply:
reason: Move this command to the Tip section (https://github.com/pingcap/docs/pull/19412/files#r1863110475), as it doesn’t need to be executed at this point.